Makefile

```diff
@@ -117,8 +121,12 @@ build-subpackages:
 	for subpackage in $(SUBPACKAGES); \
 	do \
 	    pushd $$subpackage; \
-	    $(PYTHON) ./setup.py clean --all \
-	    $(PYTHON) ./setup.py build --root=$(DESTDIR) --prefix=$(PREFIX); \
+	    $(PYTHON) ./setup.py clean --all; \
+	    if [ '$(OS_DIST)' = 'debian' ]; then \
+	        $(PYTHON) ./setup.py build; \
+	    else; \
+	        $(PYTHON) ./setup.py build --root=$(DESTDIR) --prefix=$(PREFIX); \
+	    fi; \
```

Comment:
Ah, I'm so sorry that I bothered you with this request. Please change it this to:
```
$(PYTHON) ./setup.py build; \
```
The `--root` and `--prefix` options are not supported for `build` command on any platform.

Reply:
done :smiley: